corehq/apps/reminders/views.py

```diff
@@ -102,7 +102,7 @@ def page_context(self):
         today = timezone_now.date()
 
         def adjust_next_fire_to_timezone(reminder_utc):
-            return ServerTime(reminder_utc.next_fire).user_time(timezone).done()
+            return ServerTime(reminder_utc.next_fire).user_time(timezone).done().replace(tzinfo=None)
```

Comment:
thanks, i agree that we should make it a standard that naive datetimes are always utc.

i think for this particular case though, it's probably not worth coming up with a more complex solution since this view is going away in a few months and will be replaced with a report where we do follow better timezone conventions.